README.md

```diff
@@ -40,6 +40,7 @@ OMERO.server configuration.
 
 OMERO system user, group, permissions, and data directory.
 You may need to change these for in-place imports.
+Directory paths must not have a trailing `/`
```

Comment:
It's certainly doable with a regex at the expense of making the ansible code more complicated. My thinking was that this option is unlikely to be used at the moment (mostly for the IDR), and ideally this should be an OMERO configuration property in which case this option which creates a symlink is no longer required. I can add in the regex, or if you prefer move this to "Unstable features".